Formula/toxic.rb

```diff
@@ -1,49 +1,47 @@
-require 'formula'
-
 class Toxic < Formula
-  head "git://github.com/Tox/toxic", :using => :git
+  desc "An ncurses-based Tox client"
   homepage "https://tox.chat"
+  head "git://github.com/Tox/toxic"
 
-  depends_on "libtoxcore"
-  depends_on "ncurses"
+  option "with-notifications", "Build with desktop notifications"
+
+  depends_on "pkg-config" => :build
```

Comment:
Not sure how to dependencies inn Homebrew works, but these don't match the ones listed at https://github.com/JFreegman/toxic/blob/master/INSTALL.md#dependencies

(The Python one seems to be missing at least)